dwave/system/composites/linear_ancilla.py

```diff
@@ -138,7 +138,7 @@ def sample(
 
         # Positive couplings tend to have smaller control error,
         # we default to them if they have the same magnitude than negative couplings
-        # https://docs.dwavesys.com/docs/latest/c_qpu_ice.html#overview-of-ice
+        # See the ICE documentation
```

Comment:
This quite cryptic for the uninitiated; what is ICE, one might ask. Can we provide a link to docs?

```suggestion
        # See the ICE documentation at https://docs.dwavequantum.com/en/latest/quantum_research/errors.html#ice
```